src/plugins/data/common/index_patterns/types.ts

```diff
@@ -54,6 +54,7 @@ export interface IndexPatternAttributes {
   intervalName?: string;
   sourceFilters?: string;
   fieldFormatMap?: string;
+  attributes?: { [key: string]: unknown };
```

Comment:
I think this can be defined for its current usage

Reply:
agreed, thx & done